.yarnrc.yml

```diff
@@ -16,3 +16,4 @@ unsafeHttpWhitelist:
   - localhost
 
 yarnPath: .yarn/releases/yarn-3.5.0.cjs
+checksumBehavior: "update"
```

Comment:
See this failed run https://github.com/shapeshift/web/actions/runs/6349258371/job/17247184345

Will need to re-publish all snaps packages and bump hdwallet adapter to consume these